ERCS/erc-7579.md

```diff
@@ -224,11 +234,35 @@ interface IModule {
     function onInstall(bytes calldata data) external;
     function onUninstall(bytes calldata data) external;
     function isModuleType(uint256 typeID) external view returns(bool);
+    function getModuleTypes() external view returns (uint256[]);
 }
 ```
 
 Modules MUST revert if `onInstall` or `onUninstall` was unsuccessful.
 
+
+##### Module State Management on Install and Uninstall**
```

Comment:
imo this is already sufficiently outlined in `Account configurations`

Reply:
https://github.com/zeroknots/ERC-minimalisticMSA/blob/bf1f94f03d2c1ba92670f6f7fe8805a6877c52f1/ERCS/erc-7579.md?plain=1#L118-L119 currently specifies that on uninstall, the `onUninstall` function must be called on the module - when you say that a module must specify if data is deleted on uninstall or not, where would a module specify this? in an inline comment?

Reply:
A comment would be good maybe not enough but we cannot standardize the addition of a comment 😅, but at least take into consideration and consider security measures while building modules. This note serves as a warning.